src/features/components/molecules/image-text-boxes/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { AcuityFeaturesType } from './types'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import useRegion from 'components/hooks/use-region'
+import LinkButton from 'features/components/atoms/link-button'
+
+interface ImageWithParagraphType {
+    contentData: AcuityFeaturesType
+}
+
+const ImageWithParagraph = ({ contentData }: ImageWithParagraphType) => {
+    const { is_eu } = useRegion()
+    const { mobile_flex_direction, data } = contentData
+    return (
+        <FlexBox.Box container="fluid" direction={'col'} mt={'20x'} md={{ mt: '40x' }}>
+            {data.map((feature, index) => {
+                return (
+                    <FlexBox.Box
+                        key={feature?.id}
+                        direction={mobile_flex_direction}
+                        mb={'10x'}
+                        justify={'center'}
+                        align="center"
+                        md={{
+                            direction: index == 0 || index % 2 == 0 ? 'row' : 'row-reverse',
```

Comment:
OR condition `index == 0` is not required since `0 % 2 = 0`.
```suggestion
                            direction: index % 2 == 0 ? 'row' : 'row-reverse',
```

Reply:
Fine @Nuzhy-Deriv I didn't try, good that test it